### ISSUE TITLE
Add 'moderate' link in all results view

### ISSUE THREAD

**Issue by kanghj** [TEAMMATES/repo#3019]
Presently, the instructor can see the link to moderate a student's answers only when grouping by either Giver > Question > Recipient or Giver > Recipient > Question. The instructor should be able to get to the moderations page from all results view.

**Comment by halfdreamer** [comment]
Can i try this one ...?

**Comment by kanghj** [comment]
@halfdreamer, would you like to try something easier? This might be a little too difficult and may take too long for now, as this will involve making many changes to different places.

**Comment by halfdreamer** [comment]
@kanghj  Adding moderate buttons function in Recipient->Question->Giver and Recipient->Giver->Question views seems straight forward...anyway it'll take time..I'll try something easier..

**Comment by wkurniawan07** [comment]
I tried something like this:
Question view:
![3019_1](https://cloud.githubusercontent.com/assets/7261051/6769425/fce651ca-d0d5-11e4-9e06-ac00f1284a6e.png)
RQG view:
![3019_2](https://cloud.githubusercontent.com/assets/7261051/6769426/ff08f14c-d0d5-11e4-9ddd-6a25e8e3d663.png)
The idea for the two above is the same: give the button to moderate on every single response row.
RGQ view:
![3019_3](https://cloud.githubusercontent.com/assets/7261051/6769430/292d51a2-d0d6-11e4-8638-cffc68da037e.png)
I think it's more appropriate to put the button there.
I tried on my dev server and so far all the buttons pointed to the correct page for moderating the correct person's response.
Can I proceed with this implementation?

**Comment by damithc** [comment]
To be consistent with other places, the new column can be named 'Actions'.
Even no-response rows can have the 'moderate' option, in case the
instructor wishes to enter responses on behalf of the student.

**Comment by wkurniawan07** [comment]
Good idea. After small changes:
![3019_4](https://cloud.githubusercontent.com/assets/7261051/6769597/8ed39c48-d0de-11e4-8ed5-0b2961e72c7d.png)
![3019_5](https://cloud.githubusercontent.com/assets/7261051/6769598/907bf9b4-d0de-11e4-91b1-8a04a2a60279.png)
All the buttons still pointed to the correct page for moderating the correct person's response.
I forgot to show this before, but anonymous students' responses won't be moderate-able:
![3019_6](https://cloud.githubusercontent.com/assets/7261051/6770093/6495eeec-d0f0-11e4-8042-f8461621b905.png)
And team's responses are also correctly handled, so no problem with them.
Can I proceed with this implementation?

**Comment by damithc** [comment]
@wkurniawan07 can we use the same gray color buttons as the one used in the home page? otherwise the blue button takes too much attention in this background.
@kanghj any comments?

**Comment by kanghj** [comment]
Perhaps we should show the buttons but disable them for anonymous responses? 

Since we are putting the button on every response, should we have a new issue for instructors to moderate only one question for a student? We might be able to re-use some code from feedbackQuestionSubmissionEdit.jsp.

**Comment by damithc** [comment]
Yes, 'disabled button' approach is good.
Yes, we can have it as a separate issue.

**Comment by wkurniawan07** [comment]
Changing the button color is quite straightforward to do.
![3019_7](https://cloud.githubusercontent.com/assets/7261051/6775151/dfa74e92-d167-11e4-864d-1747f1ccd418.png)
On the 'disabled button' approach, I had that in consideration as well, but in GRQ/GQR view the anonymous students' responses are not given the button to moderate as well...
![3019_8](https://cloud.githubusercontent.com/assets/7261051/6775155/f2ee857e-d167-11e4-96e8-f21f7b5a0fc8.png)
which is why I eventually opt for the design I used now, to follow suit. Of course, if the 'disabled button' approach is better, I can always change and it's quite simple to do.

**Comment by damithc** [comment]
We can do the 'disabled button' as a separate issue. It is a cosmetic thing
only.

**Comment by wkurniawan07** [comment]
We can always open those and other related ones after this is implemented. Permission to proceed :smiley:

**Comment by damithc** [comment]
Sure, proceed. Also create the related issues.

**Comment by wkurniawan07** [comment]
Will do once this implementation is merged, otherwise it doesn't really make sense.
